src/platform/ESP32/ESP32Utils.cpp

```diff
@@ -328,6 +330,62 @@ CHIP_ERROR ESP32Utils::ClearWiFiStationProvision(void)
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ESP32Utils::InitWiFiStack(void)
+{
+#if CHIP_DEVICE_CONFIG_ENABLE_WIFI
```

Comment:
Not necessary, right? It should run into this code if Wi-Fi is not enabled.

Reply:
Yes, removed it.